apps/tailwind-components/types/types.ts

```diff
@@ -110,4 +110,5 @@ export interface IInputProps {
   invalid?: boolean;
   valid?: boolean | undefined;
   disabled?: boolean | undefined;
+  required?: boolean | undefined;
```

Comment:
do we really need this at this level? I believe not?

Reply:
I'm torn on this as well. Many input are single element components. For the complex inputs where there are more the input element is nested (listbox, file input, ref, etc.), I needed a way to bind the required state to the input element. The only way was to make it a prop.

Reply:
ah, I read in the aria docs that if you have 'required' in the label text then we should NOT have also required on the input. Otherwise user will get required twice. That is why I believed we don't need it on this level anymore because we solve this central at FormField level.

Reply:
Sounds great! I've removed it.